jupyter_scheduler/models.py

```diff
@@ -192,9 +192,10 @@ class UpdateJobDefinition(BaseModel):
     tags: Optional[Tags] = None
     name: Optional[str] = None
     url: Optional[str] = None
+    schedule: Optional[str] = None
     timezone: Optional[str] = None  # Should be a timezone e.g., US/Eastern, Asia/Kolkata
     output_filename_template: Optional[str] = OUTPUT_FILENAME_TEMPLATE
-    active: bool
+    active: Optional[bool] = None
```

Comment:
It is reasonable to have a job definition without schedule because, it might be used for creating run now jobs as well. I can fix this error by not invoking task runner when there is no schedule present.

Reply:
Interesting, so job definitions can be used as both a template for "run now" and as a schedule. I'm OK with this as long as it was already agreed upon by our team.